ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template2.kt

```diff
@@ -74,21 +81,27 @@ internal fun Template2(
         PaywallBackground(state.templateConfiguration)
 
         Column {
-            Template2MainContent(state, viewModel, childModifier)
+            var packageSelectorVisible by remember {
+                mutableStateOf(state.templateConfiguration.mode != PaywallMode.FOOTER_CONDENSED)
+            }
+            Template2MainContent(state, viewModel, packageSelectorVisible, childModifier)
             PurchaseButton(state, viewModel, childModifier)
             Footer(
                 templateConfiguration = state.templateConfiguration,
                 viewModel = viewModel,
                 childModifier = childModifier,
+                allPlansTapped = { packageSelectorVisible = !packageSelectorVisible },
             )
         }
     }
 }
 
+@Suppress("LongMethod")
 @Composable
 private fun ColumnScope.Template2MainContent(
     state: PaywallState.Loaded,
     viewModel: PaywallViewModel,
+    packageSelectionVisible: Boolean,
```

Comment:
Hmm, this will be toggled when the all plans button is tapped `allPlansTapped = { packageSelectorVisible = !packageSelectorVisible },`. So, on `FOOTER_CONDENSED`, it can switch between both values